lib/commands/platform-clean.ts

```diff
@@ -0,0 +1,26 @@
+export class CleanCommand implements ICommand {
+	constructor(private $platformService: IPlatformService,
+		private $errors: IErrors) { }
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			this.$platformService.removePlatforms(args).wait();
```

Comment:
One last test would be to mocks the "removePlatform" and "addPlatform" and asserts they were executed, and were executed in that order, and the platforms passed in the args of the clean command matches the platforms passed to removePlatforms/addPlatforms methods.